src/content/docs/change-tracking/change-tracking-view-analyze.mdx

```diff
@@ -159,6 +166,34 @@ When you compare deployments:
 * Other calculations on this page will update to help you understand the difference between THIS deployment and THAT deployment. They may show increases/decreases in percentages for rates or counts (for example, +11) where it makes sense to do so.
 * You can open up to five stacked views of related deployments.
 
+### See the effects of changes on metrics and events [#trend-boards]
+
+To see how a change, such as a deployment, affects an entity's metrics and events, you can create custom trend boards. You have the option of creating up to three boards to help you make decisions about your recent change.
+
+<img
+    title="Screenshot showing two custom charts on deployments page"
+    alt="Screenshot showing two custom charts on deployments page"
+    src={trackingCustomChartsonDeploymentPage}
+/>
+<figcaption>
+    Here is an example of a single custom trend board for a deployment. Two more boards could be added to this deployment.
+</figcaption>
+
+What can you see with these custom trend boards? You can select from any metrics or events available to your entity and then apply an aggregation function to it. For example, you could make custom trend boards that answer these types of questions:
+
+* How has the average page view response time changed before and after this deployment?
+* How has the max page view response time changed before and after this deployment?
+
+Unlike regular dashboard charts, these custom trend boards are charts that are fully integrated with the time window and **compared with** settings on the **Deployment details** page. You can even export these to dashboards, and they are all tied to a specific entity and to your user account.
+
+To add a custom trend board:
+
+1. Open the **Deployments** page for your entity (for example: **[one.newrelic.com](https://one.newrelic.com/) > All Entities > (select an entity) > Deployments**).
```

Comment:
I'd remove `for example` because it's not an example, it's the way of open the page. I'd rewrite it to this: 
1. Go to **[one.newrelic.com](https://one.newrelic.com/)** > **All Entities** and select your entity.
2. Click **Deployments** in the left navigation pane.

Reply:
Thanks, Nati!

Reply:
@nbaenam thanks for the tip! I used your suggstion.